src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,227 @@
+
+Absolute/fixed positioning has been reimplemented and is more spec compliant than the old implementation. It also better matches the behavior in other browsers. The differences between the two are most visible in the areas where the old implementation did not follow the spec:
+
+- **Multi-line Inline Containing Blocks** <br>If an abspos containing block spanned multiple lines, the legacy engine might incorrectly use only a subset of the lines to compute the containing block bounds.
+- **Writing Modes** <br>The legacy engine had many problems placing out of flow elements to default position in vertical writing modes. See the next section for more details around improved writing mode support.
+
+<br>
+
+### RTL & Writing Modes
+LayoutNG was designed from the ground up to support vertical writing modes and bi-directional content. This fixes numerous issues around both writing modes, right-to-left inlines, and orthogonal flows.
+
+#### Bidirectional Text
+LayoutNG supports the most up-to-date Unicode bidirectional algorithm defined by [The Unicode Standard](https://unicode.org/standard/standard.html). Not only does it fix various cases where rendering was not correct, it also includes missing features in the old engine such as paired bracket support ([#302469](https://crbug.com/302469)).
+
+#### Orthogonal Flows
+LayoutNG improves vertical flow layout correctness, for more correct positioning of absolute positioned objects, sizing of orthogonal flow boxes especially when percent is used, and so forth. Among the 1,258 tests in W3C test suites, **103 tests that failed in the old engine pass in LayoutNG.**
+
+#### Intrinsic Sizing
+Intrinsic sizes are now calculated correctly when a LayoutNG block contains children in an orthogonal writing mode.
+
+<br>
+
+### Text Layout & Line Breaking
+The old layout engine in Chromium does text layout element by element and line by line. This has historically worked well but requires a lot of extra complexity to support complex scripts and to get good performance. It's also prone to inconsistencies in measurements which tend to manifest themselves as subtle differences in sizing of size-to-content containers and their content or unnecessary line breaks.
+
+In LayoutNG, text layout is done on a paragraph level and is then split into lines. This allows for better performance, higher quality text rendering, and more consistent line breaking. The most notable differences here from a developer and user standpoint are detailed below.
+
+#### Joining across Element Boundaries
+In some scripts graphemes join with adjacent ones and changes presentation. In LayoutNG this works even if the graphemes are in different elements, allowing the joins to be preserved even if different styling is applied ([#6122](https://crbug.com/6122)).
```

Comment:
maybe link to what a grapheme is? Or you can use the `key-term` aside to explain what the term means. It was new to me :D

Reply:
gooooooood call! dude, lovin your review suggestions. will add 👍